src/utils/uid.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+// A copy of @ckeditor/ckeditor5-utils/src/uid.js
```

Comment:
Let's add the dependency in `package.json` and use it instead of copying the util.